src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintService.java

```diff
@@ -161,10 +161,10 @@ private UUID printDocumentsWithCoversheet(FinremCaseDetails caseDetails,
 
     private BulkPrintDocument generateApplicantCoverSheet(FinremCaseDetails caseDetails, String authorisationToken) {
         CaseDocument applicantCoverSheet = coverSheetService.generateApplicantCoverSheet(caseDetails, authorisationToken);
-        log.info("Applicant cover sheet generated: Filename = {}", applicantCoverSheet.getDocumentFilename());
+        log.info("Applicant coversheet generated {} for case Id {}", applicantCoverSheet, caseDetails.getId());
```

Comment:
applicantCoverSheet is object will it print the documentFilename?

Reply:
Yes